actors/account/Cargo.toml

```diff
@@ -20,6 +20,7 @@ num-traits = "0.2.14"
 num-derive = "0.3.3"
 fvm_ipld_blockstore = "0.1.1"
 fvm_ipld_encoding = "0.2.2"
+anyhow = "1.0.56"
```

Comment:
Noooo, we're trying to kill this. It looks like it's only come in to a test, which I could believe is necessary due to existing use of anyhow. But if it's easy to avoid, I'd prefer that.

Reply:
Yeah, unfortunately it'd be a bit much to change the runtime test_utils here.